emcc.py

```diff
@@ -1021,6 +1021,11 @@ def check(input_file):
       # Apply -s settings in newargs here (after optimization levels, so they can override them)
       for change in settings_changes:
         key, value = change.split('=')
+
+        # In those settings fields that represent amount of memory, translate suffixes to multiples of 1024.
+        if key in ['TOTAL_STACK', 'TOTAL_MEMORY', 'GL_MAX_TEMP_BUFFER_SIZE', 'SPLIT_MEMORY', 'BINARYEN_MEM_MAX']:
+          value = value.lower().replace('tb', '*1024*1024*1024*1024').replace('gb', '*1024*1024*1024').replace('mb', '*1024*1024').replace('kb', '*1024').replace('b', '')
```

Comment:
Ok, changed this to a helper function in `shader.py`. The uppercase variants are supported already, since the code does `.lower()` on the input before replacing.